rows.go

```
@@ -628,15 +723,15 @@ func (f *File) DuplicateRowTo(sheet string, row, row2 int) error {
 	if row2 < 1 || row == row2 {
-		return nil
+		return err
 	}
 
 	var ok bool
 	var rowCopy xlsxRow
 
 	for i, r := range ws.SheetData.Row {
 		if r.R == row {
-			rowCopy = deepcopy.Copy(ws.SheetData.Row[i]).(xlsxRow)
+			deepcopy.Copy(&rowCopy, ws.SheetData.Row[i])
```

golangci-lint (1.64.8)

Error return value of `deepcopy.Copy` is not checked

(errcheck)